OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/ImportKeysActivity.java

```diff
@@ -208,6 +212,13 @@ protected void handleActions(@NonNull Intent intent) {
                 startListFragment(null, null, fbUsername, cloudSearchPrefs);
                 break;
             }
+            case ACTION_IMPORT_KEY_FROM_WEB_KEY_DIRECTORY: {
+                Preferences.CloudSearchPrefs cloudSearchPrefs =
```

Comment:
would be nicer to use a factory method here

Reply:
Ah, that's what you meant, I thought this was about `autovalue` only ;) Okay, let me fix that....

Reply:
Okay fixed, there are two new methods used now: `createWebKeyDirectoryOnly` and `createKeyserverOnly`.